parts/kubernetesmastercustomdata.yml

```diff
@@ -317,11 +185,19 @@ write_files:
     [Service]
     Restart=always
     ExecStartPre=/bin/mkdir -p /var/lib/kubelet
+    ExecStartPre=/bin/mount --bind /var/lib/kubelet /var/lib/kubelet
```

Comment:
Oh, it should be a merge or rebase issue. Actually I didn't change that line. What I did is just use `sed` to replace strings.